npc/other/divorce.txt

```diff
@@ -211,7 +211,8 @@ nif_in,190,112,5	script	Deviruchi#divorce	4_DEVIRUCHI,{
 		specialeffect2 EF_DEVIL;
 		wedding_sign = 0;
 		Zeny -= 2500000;
-		percentheal -100,-100;
+		percentheal -0,-100;
+		Hp -= Hp-1;
```

Comment:
How about using `Hp = 1` since you reduce the Hp to 1 only ?
as well as `Sp = 0` ?

It would avoid the "calculation" part for `Hp - 1` and more simple to understand for those who are not familiar with coding.